showcase/index.js

```diff
@@ -83,7 +81,6 @@ import TreemapExample from './treemap/dynamic-treemap';
 export const showCase = {
   ComplexChart,
   LineChart,
-  LineChartGL,
```

Comment:
maybe don't delete this demos? they are still good for testing the canvas stuff. Just remove the deck.gl content